docs/content/templates/variables.md

```diff
@@ -118,10 +121,13 @@ includes taxonomies, lists and the homepage.
 **.IsPage** Always false for nodes.<br>
 **.Site** See [Site Variables]({{< relref "#site-variables" >}}) below.<br>
 **.Hugo** See [Hugo Variables]({{< relref "#hugo-variables" >}}) below.<br>
+**.Translations** A list of translated versions of the current node. All nodes (except the pager nodes) can have translated counter parts. See [Multilingual]({{< relref "content/multilingual.md" >}}) for more info. <br>
+**.HasTranslations** Whether there are any translations to display.<br>
```

Comment:
I've tried these variables in a single template:
Line 124: .Translations gives an empty list [] instead of the expected result
Line 125: .HasTranslations gives an error: `HasTranslations is not a field of struct type *hugolib.Page`